Core/Application/CQRS/Factors/ManagersQuery.cs

```diff
@@ -47,13 +47,13 @@ private Manager GetSowing()
 
             sowing.Parameters = new List<KeyValuePair<string, string>>
             {
-                new KeyValuePair<string, string>("Date", exp.Sowing.Date.ToString("MM/dd/yyyy")),
-                new KeyValuePair<string, string>("Density", exp.Sowing.Population.ToString()),
-                new KeyValuePair<string, string>("Depth", exp.Sowing.Depth.ToString()),
-                new KeyValuePair<string, string>("Cultivar", exp.Sowing.Cultivar?.Replace('/', 'x')),
-                new KeyValuePair<string, string>("RowSpacing", exp.Sowing.RowSpace.ToString())
+                new KeyValuePair<string, string>("Date", exp.Sowing?.Date.ToString("MM/dd/yyyy")),
```

Comment:
MM/dd ??

Reply:
I really don't like it either, but I had to make that switch, because it turns out APSIM was assuming that the data was formatted like that, and was correcting it automatically. So exporting it as dd/MM actually ends up as MM/dd in the simulation and vice versa. That was a fun one to debug.